### PR TITLE
Service Account Implementation

### PR DESCRIPTION
**Pull request by future-pirate-king**

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -1291,16 +1300,75 @@
...
+    "expiryHelperText": "* This token will expire on {date}",
+    "regenerateKey": "Re-generate Key",
+    "projectAccess": "Project Access",
+    "projectAccessDescription": "These projects are applicable for the particular scope(scan) of the projects",
+    "selectedProjectAccess": "Selected - <strong>{projectAccess}</strong>",
+    "forSpecificProjects": "For Specific Projects",
+    "emptyProjectListTitle": "No project has been added",
+    "emptyProjectListDescription": "Click on \"Add Projects\" button to add the specific projects",
+    "selectedProjectListTitle": "Added Project",
+    "selectedProjectListDescription": "Lorem ipsum dolor sit amet consectetur.",
```
Comment (Yibaebi) [inline translations/en.json:1349]: Add real translation text.

File: app/components/organization/service-account/list/index.hbs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+      <Organization::ServiceAccount::Empty />
+    {{else}}
+      <div class='mb-2'>
+        <AkTable as |t|>
+          <t.head
+            data-test-serviceAccountList-thead
+            @columns={{this.columns}}
+          />
+          <t.body @rows={{this.serviceAccountList}} as |b|>
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/service-account/list/index.hbs:72]: done

File: app/components/organization/service-account/add-project/list/index.hbs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+          {{t 'noProject'}}
+        </AkTypography>
+      </div>
+    {{else}}
+      <AkTable @dense={{true}} as |t|>
+        <t.head
+          data-test-serviceAccountAddProjectList-thead
+          @columns={{this.columns}}
+        />
+        <t.body
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/service-account/add-project/list/index.hbs:62]: done

File: app/components/organization/service-account/add-project/list/index.hbs [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    {{/if}}
+  </AkPaginationProvider>
+</div>
+
+<AkStack
+  @spacing='1.5'
+  @alignItems='center'
+  @justifyContent='flex-end'
+  class='px-3 py-2'
+  local-class='api-scan-drawer-footer'
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/service-account/add-project/list/index.hbs:108]: done

File: app/components/organization/service-account/section/select-project/index.ts [hunk 5]
```diff
@@ -0,0 +1,97 @@
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+
+// eslint-disable-next-line ember/use-ember-data-rfc-395-imports
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/service-account/section/select-project/index.ts:7]: removed

File: app/components/organization/service-account/section/select-project/list/index.hbs [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+          data-test-serviceAccountSection-selectProjectList-description
+          @variant='body2'
+          @color='textSecondary'
+        >
+          {{t 'serviceAccountModule.selectedProjectListDescription'}}
+        </AkTypography>
+      </AkStack>
+
+      <AkStack @alignItems='center' @spacing='1.5'>
+        {{!-- <div local-class='search-input-container'>
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/service-account/section/select-project/list/index.hbs:67]: yes search will be supported in future

File: app/components/organization/service-account/section/select-project/list/index.ts [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+
+// eslint-disable-next-line ember/use-ember-data-rfc-395-imports
+import type DS from 'ember-data';
+import type Store from '@ember-data/store';
+import type IntlService from 'ember-intl/services/intl';
+
+import type ServiceAccountProjectModel from 'irene/models/service-account-project';
+import type ServiceAccountModel from 'irene/models/service-account';
+import parseError from 'irene/utils/parse-error';
+import ServiceAccountService from 'irene/services/service-account';
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/service-account/section/select-project/list/index.ts:16]: done

File: app/components/organization/service-account/section/select-scope/index.hbs [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+        <AkIcon @iconName='edit' />
+      </AkIconButton>
+    </AkTooltip>
+  </:headerAction>
+
+  <:default>
+    <AkCheckboxTree
+      @treeData={{this.treeData}}
+      @expanded={{this.expanded}}
+      @cascade={{true}}
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/service-account/section/select-scope/index.hbs:28]: ok removed

File: app/components/organization/settings/index.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+import Component from '@glimmer/component';
```
[the remark this answers is not in the document]
Comment (future-pirate-king) [reply app/components/organization/settings/index.js:1]: then there will be lot of undefined component errors in hbs file. That is why not converted

File: translations/en.json (hunk 1 above)
Comment (future-pirate-king) [reply translations/en.json:1349]: done

**Review by Yibaebi — APPROVED** [review]

**Pull request merged**